V2.17-RUN-16823-whats-new-only (#738)
run-ai/docs: docs/developer/overview-developer.md

---
title: Developer Documentation Overview
summary: This article is an overview of the developer documentation for the Run:ai platform.
authors:
    - Jason Novich
date: 2024-Apr-10
---

Developers can access Run:ai through various programmatic interfaces.

## API Support

The endpoints and parameters specified in the API reference are the ones that are officially supported by Run:ai. Endpoints and parameters that are **NOT** listed in the reference are not officially supported.

> The endpoints and parameters specified in the API reference are the ones that are officially supported by Run:ai. Endpoints and parameters that are **NOT** listed in the reference are not officially supported.

Reviewer: @jasonnovichRunAI it seems like this sentence is duplicated, and the sentence that was here was deleted...

Option1 : <#of versions>
Option 2: <time period>.

> For details, see the [Deprecation notifications](../home/whats-new-2-16.md#deprecation-notifications).

Author: ```suggestion
For details, see the [Deprecation notifications](../home/whats-new-2-17.md#deprecation-notifications).
```

---
title: Developer Documentation Overview
summary: This article is an overview of the developer documentation for the Run:ai platform.
authors:
    - Jason Novich
date: 2024-Apr-10
---

# Overview: Developer Documentation

Developers can access Run:ai through various programmatic interfaces. 

## API Support

> The endpoints and parameters specified in the following references are the ones that are officially supported by Run:ai. Endpoints and parameters that are **NOT** listed in the reference are not officially supported.

Reviewer: ```suggestion
The endpoints and parameters specified in the API reference are the ones that are officially supported by Run:ai. Endpoints and parameters that are **NOT** listed in the reference are not officially supported.
```

Endpoints and parameters marked as deprecated means they remain operational and can be used; however, Run:ai will **NO LONGER RECOMMEND USING THEM**, **WILL NOT ADD FUNCTIONALITY TO THEM**, and **WILL NO LONGER BE SUPPORTED AFTER**

Option1 : <#of versions>

> Option 2: <time period>. 

Reviewer: @jasonnovichRunAI keeping an open comment here until we have a decision on the timeframe

Author: ```suggestion
For details, see the [Deprecation notifications](../home/whats-new-2-17.md#deprecation-notifications).
```

> For details, see the [Deprecation notifications](../home/whats-new-2-16.md#deprecation-notifications).

Reviewer: @jasonnovichRunAI I see that this is a link to 2.16 what's new. Maybe we should say something like - "see the deprecation notification of the relevant version in the what's new article", and link to the latest what's new?

Author: fixed to 2.17.
